src/css/bootstrap-overrides.scss

```diff
@@ -181,7 +181,6 @@ $input-icon-height: 20px;
 
 .modal {
   .modal-dialog {
-    margin: 80px 32px 0;
```

Comment:
I fixed that problem in my branch. Just replace that by `margin-top: 80px` or don't touch it at all